catalog/spatial/wfs/spatial-wfs-featuretransformer-xstream/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/xstream/XStreamWfs11FeatureTransformer.java

```diff
@@ -50,7 +52,9 @@
 
   @Override
   public Optional<Metacard> apply(InputStream document, WfsMetadata<FeatureTypeType> metadata) {
-    XStream xStream = new XStream(new WstxDriver());
+    final WstxDriver driver = new WstxDriver();
+    XStream xStream =
+        AccessController.doPrivileged((PrivilegedAction<XStream>) () -> new XStream(driver));
```

Comment:
❓ Is there a way to write tests for this?

❓ Is there a better way to do this? The `spatial-wfs-featuretransformer-xstream` bundle already has permission to `createClassLoader`: https://github.com/codice/ddf/blob/0256aa6905e9f817d43b948811485c059231eefe/distribution/ddf-common/src/main/resources/security/default.policy#L125-L127.

Reply:
> Is there a better way to do this? The spatial-wfs-featuretransformer-xstream bundle already has permission to createClassLoader:

Since it already had the permission and adding the doPrivileged call fixed it, it must be a bundle earlier in the call stack that is missing the permission. Depending on the bundle, it may make sense to grant it the permission instead of doing doPrivileged (e.g. I'd probably lean towards granting camel bundles the permission because somebody creating a new route down the road might run into this again).

Regarding testing, an itest would catch missing permissions